src/App.jsx

```diff
@@ -14,33 +16,89 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App = () => (
-  <div className="section content">
-    <div className="buttons">
-      <button type="button" className="button is-info is-light">
-        Sort alphabetically
-      </button>
 
-      <button type="button" className="button is-success is-light">
-        Sort by length
-      </button>
+const sortAlphabetical = 'name';
+const sortLength = 'length';
 
-      <button type="button" className="button is-warning is-light">
-        Reverse
-      </button>
 
-      <button type="button" className="button is-danger is-light">
-        Reset
-      </button>
-    </div>
+export const App = () => {
+  const [sortField, setSortField] = useState('');
+  const [isReversed, setIsReversed] = useState(false);
+  const [isChanged, setIsChanged] = useState(false);
```

Comment:
why do you need this state?
Wnen state changes - component will rerender without additional logic.